fury/shaders/base.py

```diff
@@ -200,6 +221,50 @@ def cbk(caller, event, calldata=None):
     return id_observer
 
 
+def shader_apply_effects(
+        showm, actor, effect=None, effects=None, priority=0):
+    """This applies a specific opengl state (effect) or a list of effects just
+    before the actor's shader be executed.
+
+    Parameters
+    ----------
+    showm : fury.window.ShowManager
+    actor : vtk actor
+    effect :  function
+        a function with a glState as argument
+    effects : a list of functions, optional
```

Comment:
confusing to have `effects` and `effect`. I believe it can be only one argument. You can use `isinstance(object, Iterable) or isinstance(object, Callable)` to difference them

Reply:
Ok. I think that it's better.

Reply:
Done